backend/user_management/minio/routers/upload.py

```
@@ -1,41 +1,152 @@
+        file.file.seek(0)
+
+        if file_type in ["profile", "product photo"] and content_type not in ALLOWED_IMAGE_TYPES:
+            raise HTTPException(status_code=400, detail=f"Invalid image file type: {content_type}")
+        if file_type == "product video" and content_type not in ALLOWED_VIDEO_TYPES:
+            raise HTTPException(status_code=400, detail=f"Invalid video file type: {content_type}")
+        if file_size > MAX_FILE_SIZE_MB * 1024 * 1024:
+            raise HTTPException(status_code=400, detail=f"File size exceeds {MAX_FILE_SIZE_MB}MB")
+
+
+@router.patch("/upload", response_model=ProductPhotoUploadResponse)
+async def update_uploaded_files(
```

Ruff (0.8.2)

Missing return type annotation for public function `update_uploaded_files`

(ANN201)